### PR TITLE
fixed missing hartsel reference when determining HARTSELLEN

### PR DESCRIPTION
**Pull request by stefano-codasip**
There is a paragraph in section 3.3 that explains how to determined HARTSELLEN. However, the word "hartsel" has been left out, and the sentence no longer makes sense. The fix is for adding "hartsel" back.

### CODE REVIEW

**Review by JanMatCodasip — APPROVED** [review]
Thank you for filing this PR with the fix.

**Comment by stefano-codasip** [comment]
@pdonahue-ventana this PR requires a further approval for the Workflow 
image

**Comment by pdonahue-ventana** [comment]
I normally wait for @rtwfroody to do the final merge.

**Pull request merged**
